vulcanai2/models/cnn.py

```diff
@@ -92,8 +94,42 @@ def __init__(self, name, in_dim, config, save_path=None,
             lr_scheduler, early_stopping, criter_spec)
 
     def _create_network(self, **kwargs):
-
+        self._in_dim = self.in_dim
         conv_hid_layers = self._config.units
+
+        if len(self.in_dim) > 1:
+            # Calculate converged in_dim for the MultiInput ConvNet
+            dim_tmp = []
+            # Create empty input tensors
+            in_tensors = [torch.empty(x) for x in self.in_dim]
+            # Calculate the number of elements in each input tensor 
+            els = [t.numel() for t in in_tensors]
```

Comment:
this will take into account the batch size as well. Are you sure numel is what you want to use?